### PR TITLE
chore(e2e): Create an index and drop it COMPASS-5469

### PR DESCRIPTION
**Pull request by lerouxb**
Just this one basic test for indexes for now. We can probably test out the options and multi-key indexes when we redo that screen.

### CODE REVIEW

File: packages/compass-e2e-tests/helpers/commands/add-collection.ts [hunk 1]
```diff
@@ -98,6 +98,7 @@ export async function addCollection(
     );
     await menu.waitForDisplayed();
     const span = await menu.$(`span=${options.timeseries.granularity}`);
+    await span.waitForDisplayed();
```
Comment (lerouxb) [inline packages/compass-e2e-tests/helpers/commands/add-collection.ts:101]: Drive-by for something I missed in the previous PR.

**Review by addaleax — APPROVED** [review]

**Comment by lerouxb** [comment]
got the windows npm install flake. will have to re-run check and test.
